### PR TITLE
Revise 2.16 for clarity. Fixes #102.

### LINKED ISSUE #102: 2.16 (relaxed-sop) what are the bullet items?
They need some text explaining them.  Perhaps they are "examples"?

### PR DESCRIPTION
**Pull request by hober**
***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/pull/117.html" title="Last updated on Feb 2, 2021, 12:25 AM UTC (dedf725)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/117/48c8ce7...dedf725.html" title="Last updated on Feb 2, 2021, 12:25 AM UTC (dedf725)">Diff</a>

### CODE REVIEW

**Comment by pes10k** [comment]
This looks great to me.  Other relevant examples here might be referrer policy, or cookies, where on some platforms sites can relax (or remove) privacy protections recently achieved with stronger defaults.  

I don't mean to clutter the document or belabor a point though.  If you think more text / the above examples would be helpful, im happy to add to yr PR.  Otherwise, whats there also looks great!

File: index.bs [hunk 1]
```diff
@@ -695,20 +695,50 @@ your specification in an outlier, justifying that claim is in
...
+[[PERMISSIONS-POLICY]] defines a mechanism
+that can be used by sites to prevent untrusted <{iframe}>s from using such a feature.
+
+<div class=example>
+The {{Document/domain|document.domain}} setter can be used to relax the [=same-origin policy=].
+The most effective mitigation
+would be to simply remove it from the platform altogether (see [[#drop-feature]]),
+though its widespread usage
+suggests an [iterative approach to removal](https://github.com/mikewest/deprecating-document-domain/)
+may be more prudent.
```
[the remark this answers is not in the document]
Comment (atanassov) [reply index.bs:721]: The comment was about: 

> widespread usage suggests an [iterative approach to removal](https://github.com/mikewest/deprecating-document-domain/) may be more prudent.

**Review by atanassov — APPROVED** [review]
LGTM with the added nits addressed.

**Review by rhiaro — APPROVED** [review]
Looks good to me, agree with @atanassov's comments.

File: index.bs [hunk 2]
```diff
@@ -695,20 +695,50 @@ your specification in an outlier, justifying that claim is in
 order.
 
 <h3 class=question id="relaxed-sop">
-  Do features in your specification enable downgrading default security
-  characteristics?
+  Do features in your specification enable sites to downgrade default security
```
[the remark this answers is not in the document]
Comment (hober) [reply index.bs]: Changed to "origins" in f68dab5.

File: index.bs (hunk 1 above)
[the remark this answers is not in the document]
Comment (hober) [reply index.bs]: I tried shortening this in f68dab5.

**Comment by hober** [comment]
> This looks great to me. Other relevant examples here might be referrer policy, or cookies, where on some platforms sites can relax (or remove) privacy protections recently achieved with stronger defaults.
> 
> I don't mean to clutter the document or belabor a point though. If you think more text / the above examples would be helpful, im happy to add to yr PR. Otherwise, whats there also looks great!

I think it would be helpful to add more examples, though perhaps that's best done in a followup PR.

**Pull request merged**